scripts/import_pressbooks.py

```diff
@@ -0,0 +1,149 @@
+"""
+To run:
+
+PYTHONPATH=. python ./scripts/import_pressbooks.py /olsystem/etc/openlibrary.yml ./path/to/pressbooks.json
+"""
+
+import json
+import datetime
+import logging
+import requests
+
+from infogami import config  # noqa: F401
+from openlibrary.config import load_config
+from openlibrary.core.imports import Batch
+from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
+
+logger = logging.getLogger("openlibrary.importer.bwb")
+
+
+langs = {
+    lang['identifiers']['iso_639_1'][0]: lang['code']
+    for lang in requests.get(
+        'https://openlibrary.org/query.json',
+        {
+            'limit': '500',
+            'type': '/type/language',
+            'identifiers.iso_639_1~': '*',
+            'identifiers': '',
+            'code': '',
+        },
+    ).json()
+}
+
+
+def convert_pressbooks_to_ol(data):
+    book = {"source_records": ['pressbooks:%s' % data['url']]}
+    if data.get('isbn'):
+        book['isbn_13'] = [
+            isbn.split(' ')[0].replace('-', '') for isbn in data['isbn'].split('; ')
+        ]
+    if data.get('name'):
+        book['title'] = data['name']
+    if data.get('languageCode'):
+        book['languages'] = [langs[data['languageCode'].split('-', 1)[0]]]
+    if data.get('author'):
+        book['authors'] = [{"name": a} for a in data.get('author')]
```

Comment:
```suggestion
    if name := data.get('name'):
        book['title'] = name
    if language_code := data.get('languageCode'):
        book['languages'] = [langs[language_code.split('-', 1)[0]]]
    if authors := data.get('author'):
        book['authors'] = [{"name": author} for author in authors]
```

Reply:
Actually because python doesn't have scoped variables, this would cause all the variables to leak out into the main function body scope, which could cause annoying little bugs since we have so many of these if statements. Opting to leave as is for now